pkg/testcase/node.go

```diff
@@ -36,7 +36,7 @@ func TestNodeStatus(
 				nodeAssertVersion(g, node)
 			}
 		}
-	}, "1500s", "20s").Should(Succeed())
+	}, "2100s", "20s").Should(Succeed())
```

Comment:
can we change to 10s the second arg? sometimes when its already ok waiting another ~20s is to much

Reply:
resolved